core/src/main/java/com/microsoft/applicationinsights/channel/concrete/inprocess/InProcessTelemetryChannel.java

```diff
@@ -227,6 +238,8 @@ public synchronized void stop(long timeout, TimeUnit timeUnit) {
             telemetriesTransmitter.stop(timeout, timeUnit);
             stopped = true;
         } catch (Throwable t) {
+            InternalLogger.INSTANCE.error("Exception generated while stopping telemetry transmitter");
+            t.printStackTrace();
```

Comment:
"printStackTrace()" writes to std error regardless of whether or not SDKLogger is in use. This may not be desirable. Ideally we print these only when SDKLogger is present.

Reply:
@grlima  Yes printStackTrace() writes to std error stream. It might overflood incase there are too many exceptions. However, my concern here was if there are deep stack traces they might get trimmed by the SDKLogger which is not desirable. I can double check that behavior and if it's not true then I can pass stackTrace as an argument to the logger.